infrastructure/amplitude/segment_plugin.go

```diff
@@ -0,0 +1,90 @@
+package amplitude
```

Comment:
This seems to be missing tests?

Reply:
`client_test.go` encompasses tests for this file, didn't want to introduce another set of same tests here.

Reply:
Doesn't client_test use fakesegmentclient?

Reply:
Correct, with the Amplitude client [wiring the Segment plugin](https://github.com/snyk/snyk-ls/blob/2ed76537e9ba1fe39da665e5ec2386ce924f97a1/infrastructure/amplitude/client.go#L56). I don't want to test Amplitude internals here, but if you have some specific cases you can think of that would break, please let me know.

Reply:
[This test](https://github.com/snyk/snyk-ls/blob/2ed76537e9ba1fe39da665e5ec2386ce924f97a1/infrastructure/amplitude/client_test.go#L68) exactly tests if an event gets enqueues with a user id. It uses a real Amplitude client with Segment plugin and mock Segment API.

Segment plugin is just a dumb plugin wrapping Segment API.